lib/transaction.go

```diff
@@ -177,10 +177,10 @@ func (o *Transaction) Sign(kp keypair.KP, networkID []byte) {
 }
 
 // NextSourceCheckpoint generate new checkpoint from current Transaction. It has
-// 2 part, "<head>-<body>".
+// 2 part, "<subtracted>-<added>".
 //
-// <head>: hash of last paid transaction of source account
-// <body>: hash of last transaction
+// <subtracted>: hash of last paid transaction, it means balance is subtracted
+// <added>: hash of last added transaction, it means balance is added
```

Comment:
Suggestion: We could use "deposit" and "withdrawal" instead. That would be in line with the name introduced in `account.go` and those terms are used commonly in association with bank account.

Reply:
@Geod24 I like new terms. :)

Reply:
@Geod24 Anyway, if we limit the feature of impl. to the payment, it will be good, but if the transaction will be not used only for payment, bank related terms is not suitable, because current transaction structure can contain the other type of operations.

Reply:
What kind of other operations do you envision ? And if we do, won't we run into the same problem as we do now ?

Reply:
maybe voting?